jit-service/src/main/java/org/dcsa/jit/service/OperationsEventService.java

```diff
@@ -29,14 +42,40 @@ public static class OperationsEventFilters {
     String unLocationCode;
     String facilitySMDGCode;
     String operationsEventTypeCode;
-    String eventCreatedDateTime;
+    List<ParsedQueryParameter<OffsetDateTime>> eventCreatedDateTime;
     String sort;
     Integer limit;
     String cursor;
     String apiVersion;
   }
 
-  public List<OperationsEventTO> findAll(OperationsEventFilters operationsEventFilters) {
-    return eventRepository.findAll().stream().map(operationsEventMapper::toTO).toList();
+  public List<OperationsEventTO> findAll(
+      HttpServletRequest request, final OperationsEventFilters requestFilters) {
+
+    Paginator paginator = new Paginator(objectMapper);
```

Comment:
Why are you instantiating a new one. Paginator is a `@Component` you should be able to just inject it (at least if Application `@ComponentScan`s the entire org.dcsa package).

Reply:
Done